Casks/i/ijhttp-cli.rb

```diff
@@ -0,0 +1,21 @@
+cask "ijhttp-cli" do
+  version "243.24978.46"
+  sha256 "2fdbbf634a43fce0f6236597ea6815e6b88ff32ec893eeb355c33f59926ceeb1"
+
+  url "https://download.jetbrains.com/resources/intellij/http-client/#{version.major_minor_patch}/intellij-http-client.zip"
+  name "IntelliJ HTTP Client CLI"
+  desc "HTTP client from JetBrains IDEs available as a standalone CLI tool"
+  homepage "https://www.jetbrains.com/ijhttp/download"
+
+  livecheck do
+    url "https://jb.gg/ijhttp/latest"
+    regex(/(\d+(?:\.\d+)+)/i)
+    strategy :header_match # the url redirects us to archive download, so :page_match won't work
```

Comment:
```suggestion
    strategy :header_match
```

The default regex should work here, so we don't need to specify one.
Also, the comment isn't necessary as the use of the strategy is self-documenting.

Reply:
You're right -- thanks!